Product/DBScripts/nhincdb/nhincdb.sql

```diff
@@ -474,6 +466,14 @@ CREATE TABLE IF NOT EXISTS patientdb.phonenumber (
         ON UPDATE NO ACTION
 ) COMMENT = 'Phone Numbers';
 
+CREATE TABLE IF NOT EXISTS patientdb.recordlocatorservice (
```

Comment:
Update this in oracle scripts as well

Reply:
Oracle-script had no affect on this change